script/Arcueid/Day 7/07_00_ARC07_3D.txt

```diff
@@ -329,7 +331,8 @@ Strength drains out of me, and I feel as though I'm about to collapse to my knee
 [sha:f2dc2e6c45f1cc51d1826a1beec849bd58aa598f]{
 -- Page 253, Offset 17186.
 -- 虚脱感と自己嫌悪がぐるぐると回っている。
-My mind churns with a mixture of despondency and self-loathing.
+//despondancy is an odd word, also want two nouns
+My mind churns with a mixture of melancholy and self-loathing.
```

Comment:
This one's pretty literally despondency according to the dictionary. I might keep it like that since I think despondent has more of a hopeless vibe (what was even the point of killing vlov), whereas melancholy is a more generic sadness

Reply:
I concur, I think we should revert. I don't necessarily care about the dictionary definition of 虚脱感, but I do agree that despondency is a little more precise.